### PR TITLE
Various clean ups

### PR DESCRIPTION
**Pull request by petter-fidesmo**
This solves #29 #30

### CODE REVIEW

**Comment by petter-fidesmo** [comment]
@Franzaine What do you think of these updates?

File: core/src/main/java/nordpol/Apdu.java [hunk 1]
```diff
@@ -108,7 +107,8 @@ public static String encodeHex(byte b) {
      * @return true if the APDUs match, false if not
      */
     public static boolean hasStatus(byte[] receivedApdu, byte[] statusCode) throws IOException {
-        return hasStatus(encodeHex(receivedApdu), encodeHex(statusCode));
+        byte[] receivedStatus = statusBytes(receivedApdu);
+        return receivedStatus[0] == statusCode[0] && receivedStatus[0] == statusCode[0];
```
Comment (Franzaine) [inline core/src/main/java/nordpol/Apdu.java]: my guess is that this should be:
```return receivedStatus[0] == statusCode[0] && receivedStatus[1] == statusCode[1];```

**Comment by Franzaine** [comment]
@petter-fidesmo I think this looks great! Do you think that anybody will use ```transceiveAndGetResponse```?
I'm having a hard time understanding the tests completely but they seem to pass which I guess is a... good sign?

File: core/src/main/java/nordpol/Apdu.java (hunk 1 above)
Comment (petter-fidesmo) [inline core/src/main/java/nordpol/Apdu.java]: @Franzaine Thanks, a bit of an epic fail one could say... It's also clear that the unit tests are lacking not catching this :-)

**Comment by petter-fidesmo** [comment]
@Franzaine I think it could be useful, and since there is already a user (OTP), I think we can leave it. Just removed the "generic" one since there are several other things that needs to be handled for that to work (e.g. the class and Le of the GET RESPONSE).

**Comment by Franzaine** [comment]
:+1:

**Pull request merged**